docs/tf_model_binary_input.md

```diff
@@ -0,0 +1,98 @@
+# TF model conversion to IR with scaling for binary input
+
+This document guides how to prepare TensorFlow model to use binary input with OpenVINO&trade; Model Server.
+
+- In this example TF model that will be converted is [ResNet in TensorFlow](https://github.com/tensorflow/models/tree/v2.2.0/official/r1/resnet).
+
+- To convert TensorFlow model into Intermediate Representation format model_optimizer tool can be used. There are several ways to store TensorFlow model and in this guide we are going to convert SavedModel format. More information about conversion process can be found on the openVINO&trade; documentation in [Converting a TensorFlow* Model](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow.html#savedmodel_format) guide.
+
+- Sending requests with binary input requires some adjustments in deployment for both OVMS and model. More informations can be found on [Support for binary inputs data in OpenVINO Model Server](https://github.com/openvinotoolkit/model_server/blob/main/docs/binary_input.md).
+
+## Steps
+
+### Prepering the Model
+
+Download the model
+```Bash
+wget http://download.tensorflow.org/models/official/20181001_resnet/savedmodels/resnet_v2_fp32_savedmodel_NHWC.tar.gz
+
+tar -xvzf resnet_v2_fp32_savedmodel_NHWC.tar.gz 
+
+mv resnet_v2_fp32_savedmodel_NHWC/1538687283/ resnet_v2
+rmdir resnet_v2_fp32_savedmodel_NHWC/
+```
+*Note:* Directories operations are not neccessery for the prepartion, but in this guide in order to keep the commands as simple as possible the directories are simplified.
+
+Pull the latest openvino ubuntu_dev image from dockerhub
+```Bash
+docker pull openvino/ubuntu18_dev:latest
+```
+
+Convert the TensorFlow model to Intermediate Representation format using model_optimizer tool:
+```Bash
+docker run -u $(id -u):$(id -g) -v `pwd`/resnet_v2/:/resnet openvino/ubuntu18_dev:latest deployment_tools/model_optimizer/mo.py --saved_model_dir /resnet/ --output_dir /resnet/IR/1/ --input_shape=[1,224,224,3] --mean_values=[123.68,116.78,103.94] --reverse_input_channels
+```
```

Comment:
it would be good to explain the source of the mean values and add reference to TF preprocessing. Add info that some models might require --scale parameter